src/main/kotlin/org/stellar/walletsdk/utils/accountAvailableNativeBalance.kt

```diff
@@ -0,0 +1,14 @@
+package org.stellar.walletsdk.utils
```

Comment:
The Kotlin [convention of source files ](https://kotlinlang.org/docs/coding-conventions.html#source-file-names)starts with capital letter.

Reply:
@lijamie98, do you mean we should rename `utils` to `Utils` or all the file names should start with a capital letter? `utils` is a package inside `org.stellar.walletsdk`, so I used a lowercase name there. All the helper methods that don't have a class or interface are lowercase as well. I'm still learning Kotlin conventions, so I'm happy to change anything that is off. :sweat_smile:

Reply:
Oh, what I meant is the file name `accountAvailableNativeBalance.kt` should be renamed to `AccountAvailableNativeBalance.kt`. 

When we rename the file, the name is different from the function. That is because Kotlin/Java developers tends to aggregate a couple of functions in one file.

For example, you may create the file `AccountHelper.kt` and group these functions in the same file. 
- accountAvailableNativeBalance
- accountReservedBalance
- fetchAccountFromAddress